package.json

```diff
@@ -620,6 +620,14 @@
 				{
 					"command": "cosmosDB.deleteStoredProcedure",
 					"when": "never"
+				},
+				{
+					"command": "cosmosDB.executeAllMongoCommands",
+					"when": "never"
+				},
```

Comment:
I find it sensible to run all the commands using the command palette. I prefer the redundancy against codelens momentarily failing.
Given that (currently) errors in parsing can prevent codelens from appearing, and that codelens is not very well tested, I'd vote for having this function included. :)

Reply:
We don't need to enable them based on the fact that CodeLens might or might not fail. If that happens, it should be investigated.

You can still do this via the keyboard shortcut.

Personally I don't have a strong opinion on this, though.

Reply:
I strongly feel that "cosmosDB.executeMongoCommand" should _not_ be in the command palette because you lose context of your cursor when you open the command palette and it always runs the first command.

I don't have a strong opinion on "cosmosDB.executeAllMongoCommands" since the cursor doesn't matter in that case.

Reply:
cosmosDB.executeMongoCommand from the palette does not lose the cursor context - it runs the command the cursor is on.

I've added both commands back but set them to only be visible when editorLangId=='mongo'.

I'll check it in this way but feel free to come by and discuss further.